### PR TITLE
fix(client): fix dynamic key parsing

### PR DESCRIPTION
**Pull request by fortuna**
Fix regression introduced in https://github.com/Jigsaw-Code/outline-apps/pull/2192.

### CODE REVIEW

File: client/src/www/app/outline_server_repository/server.ts [hunk 1]
```diff
@@ -135,7 +136,7 @@ export class OutlineServer implements Server {
   }
 }
 
-function parseTunnelConfigJson(responseBody: string): TunnelConfigJson | null {
+export function _parseTunnelConfigJson(responseBody: string): TunnelConfigJson | null {
```
Comment (jyyi1) [inline client/src/www/app/outline_server_repository/server.ts:139]: Is it OK to export a function starting with `_`?

File: client/src/www/app/outline_server_repository/server.ts [hunk 2]
```diff
@@ -145,8 +146,17 @@ function parseTunnelConfigJson(responseBody: string): TunnelConfigJson | null {
     );
   }
 
+  const transport: TransportConfigJson = {
+    host: responseJson.server,
```
Comment (jyyi1) [inline client/src/www/app/outline_server_repository/server.ts:150]: Why couldn't we just change `host` to `server`?

**Review by jyyi1 — COMMENTED** [review]
Thanks for the fix!

File: client/src/www/app/outline_server_repository/server.ts (hunk 1 above)
Comment (fortuna) [reply client/src/www/app/outline_server_repository/server.ts]: I was trying to keep the visibility protected, but the style guide doesn't actually allow that, but I found a better [recommendation](go/typescript-testing#export-visibility).

**Review by jyyi1 — APPROVED** [review]

**Pull request merged**